packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -1,11 +1,6 @@
-# Apex Replay Debugger for Visual Studio Code (Beta)
```

Comment:
@ntotten - Sorry, should've thought of this earlier. Isn't Apex Replay Debugger still in beta for now? If so, we should leave in ` (Beta)` and the legal disclaimer.